modules/about-etcd-encryption.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * authentication/encrypting-etcd.adoc
+
+[id="about-etcd_{context}"]
```

Comment:
s/about-etcd/about-tetcd-encryption

For [modules](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata), the anchor needs to match the file name.

Please update the other modules, too.